src/assets/scripts/client/math/flightMath.js

```diff
@@ -203,8 +203,9 @@ export const calculateTurnInitiaionDistance = (aircraft, currentWaypointPosition
     const nominalBankAngleDegrees = 25;
     const speed = kn_ms(aircraft.speed);
     const bankAngle = degreesToRadians(nominalBankAngleDegrees);
+    const nextWaypointModel = aircraft.fms.currentLeg.nextWaypoint;
```

Comment:
Yeah that second condition won't be covered by this. :(

Really what I should have done was make a new `getNextWaypointModel`. Because we _do not_ want only the position model, due to our attempt to check `model.isVector` below, which can't be done with the position alone.